app.json

```diff
@@ -83,6 +83,7 @@
           }
         }
       ],
+      "expo-asset",
```

Comment:
I really have no clue, tried a lot but anything I did did not resolve this warning cause we have no single hooks property in the app.json
and it still works perfect, therefore I left it as it is, maybe ist some expo-doctor issue and not an issue with our code

Reply:
The cause of the problem is that in `app.config.js`, we add the hooks object to config. But hooks are no longer supported by expo. That's why I updated sentry's integration. It would be good if you take a look at this commit. @donni106 

https://github.com/smart-village-solutions/smart-village-app-app/pull/1142/commits/981466e38a85970b1c6dbfe1ab31b0b464acebe8